de/privacy/current.md

```diff
@@ -109,16 +111,6 @@ Sofern Sie gleichzeitig bei Google eingeloggt sind, ist der Google-Service in de
 
 Weitere Informationen von Google über den Umgang mit Nutzerdaten (Datenschutzerklärung) erhalten Sie auf: [http://www.google.com/intl/de/policies/privacy](http://www.google.com/intl/de/policies/privacy)
 
-#### Google Web Fonts
-
-Diese Seite nutzt zur einheitlichen Darstellung von Schriftarten so genannte Web Fonts, die von Google bereitgestellt werden. Beim Aufruf einer Seite lädt Dein Browser die benötigten Web Fonts in Deinen Browsercache, um Texte und Schriftarten korrekt anzuzeigen.
-
-Zu diesem Zweck muss der von Dir verwendete Browser Verbindung zu den Servern von Google aufnehmen. Hierdurch erlangt Google Kenntnis darüber, dass über Deine IP-Adresse unsere Website aufgerufen wurde. Die Nutzung von Google Web Fonts erfolgt im Interesse einer einheitlichen und ansprechenden Darstellung unserer Online-Angebote. Dies stellt ein berechtigtes Interesse im Sinne von Art. 6 Abs. 1 lit. f DSGVO dar.
-
-Wenn Dein Browser Web Fonts nicht unterstützt, wird eine Standardschrift von Deinem Computer genutzt.
-
-Weitere Informationen zu Google Web Fonts findest Du unter [hier](https://developers.google.com/fonts/faq) und in der Datenschutzerklärung von Google, [hier](https://policies.google.com/privacy?hl=de).
-
 #### Google reCAPTCHA
```

Comment:
You can also remove this. I don't know from any part at serlo.org where we use google recaptcha